scripts/types_generator/extractor.py

```diff
@@ -27,12 +29,13 @@ def extract_endpoints(api: Schema):
             for value in list(properties.get("parameters", [])):
                 assert isinstance(value, dict)
                 if value.get("in") == "query" and value.get("name") == "version":
+                    version: datetime.date = value.get("schema", {}).get("enum")[0]
                     endpoints.append(
                         ApiEndpoint(
                             class_name=to_classname(f"{path_formatted}_{http_method.title()}_endpoint", True),
                             path=path,
                             method=str(http_method_raw).upper(),
-                            version=value.get("schema", {}).get("enum")[0],
+                            version=version.isoformat(),
```

Comment:
This breaks when I call `poe schema` against prod. Did you forget to parse it to date on line 32?

```
[2024-02-07 10:23:55][INFO][__main__][run]: Saving extracted endpoints information to '/Users/radek/GIT/ibm-generative-ai/src/genai/schema/_endpoints.py'
[2024-02-07 10:23:55][ERROR][__main__][run]: 'str' object has no attribute 'isoformat'
Traceback (most recent call last):
  File "/Users/radek/GIT/ibm-generative-ai/scripts/types_generator/main.py", line 99, in run
    for model in extract_endpoints(schema)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/GIT/ibm-generative-ai/scripts/types_generator/extractor.py", line 38, in extract_endpoints
    version=version.isoformat(),
            ^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'isoformat'
```

Reply:
it works after the update!